docs/copilot/ai-powered-suggestions.md

```diff
@@ -0,0 +1,208 @@
+
+The following tutorials walk you through using GitHub Copilot in VS Code.
+
+1. [Getting started](/docs/copilot/getting-started.md): set up VS Code for Copilot and get your first AI-powered suggestions
+
+1. [Getting started with Copilot Chat](/docs/copilot/getting-started-chat.md): install the Copilot Chat extension and have your first AI chat conversations
+
+## Using Copilot
+
+Now that you've signed up for Copilot and activated the extension, let's see its assistance in action!
+
+GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks, and it works especially well for Python, JavaScript, TypeScript, Ruby, Go, C# and C++.
+
+## Inline suggestions
+
+Copilot presents suggestions automatically in the editor to help you code more efficiently. There are just three steps to harnessing these suggestions:
+
+1. Start writing code (or code-related items, like comments or tests).
+
+    Copilot can provide suggestions for various languages and frameworks. For any given input, Copilot might offer multiple suggestions. You can select which suggestion to use, or reject all suggestions.
+
+1. Receive a Copilot suggestion in gray ghost (faded) text.
+
+    Ghost text is placeholder text that will be replaced by input you type or select from Copilot.
+
+    As an example, in a JavaScript file, you can type the following function header:
+
+    ```js
+    function calculateDaysBetweenDates(begin, end) {
+    ```
+
+    Copilot will provide a suggestion like in the following example:
+
+    ![JavaScript ghost text suggestion](images/overview/js-suggest.png)
+
+1. Choose to accept Copilot's suggestion.
+
+    For any given input, Copilot might offer multiple suggestions. When Copilot offers a suggestion, you can accept it with the `kbstyle(Tab)` key, or hover over the suggestion to see the inline suggestion toolbar:
+
+    ![JavaScript ghost text suggestion](images/overview/copilot-hover-highlight.png)
+
+    In the previous image, Copilot presents three suggestions. You can accept the entire suggestion with `kbstyle(Tab)`, or only part of the suggestion with `kbstyle(Ctrl+RightArrow)`. You can switch between suggestions in the suggestion toolbar, or use the keyboard shortcut `kb(editor.action.inlineSuggest.showNext)` instead.
+
+    If you don't want to accept any of the suggestions, you can continue typing, and Copilot will continue providing suggestions as you work.
+
+## Chat features
+
+In addition to inline suggestions, you can also get assistance from Copilot via a chat interface. Copilot Chat is supported in several ways:
+
+* **Chat view:** Ask Copilot for help with any task or question in the GitHub Copilot Chat view.
+* **Inline Chat:** Talk with Copilot while writing code, inline in your files.
+* **Quick Chat:** Bring up a Chat dropdown for quick questions and suggestions.
+
+### Install the GitHub Copilot Chat extension
```

Comment:
The Chat extension comes as part of Copilot.
I am not sure if we need to call this out. This extension is an implementation detail. Maybe we could remove this section. So my point is that all Copilot users will already have it installed.

fyi @bamurtaugh

Reply:
I think calling this out as an implementation detail is a good idea - there's been user feedback that it's confusing why there are multiple extensions. But I agree with what @isidorn mentions that maybe this shouldn't be an entire section, just a note in one of the earlier sections that installing the main Copilot extension will give you both Copilot completions and Copilot Chat.